internal/mode/static/manager.go

```diff
@@ -223,3 +255,21 @@ func prepareFirstEventBatchPreparerArgs(
 
 	return objects, objectLists
 }
+
+func setInitialConfig(
+	reader client.Reader,
+	logger logr.Logger,
+	eventRecorder record.EventRecorder,
+	logLevelSetter ZapLogLevelSetter,
+	configName types.NamespacedName,
+) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	var config nkgAPI.NginxGateway
+	if err := reader.Get(ctx, configName, &config); err != nil {
+		return err
+	}
+
+	return updateControlPlane(&config, logger, eventRecorder, configName, logLevelSetter)
```

Comment:
we intentionally don't set status here, right? 
because the status updater uses the cache which has not been started yet.
I suggest we document this "why" here, if that is the case